### ISSUE TITLE
Home feed does not load

### ISSUE THREAD

**Issue by silverandgray** [tootsuite/mastodon#7435]
My home feed loads on mastodon.social but not on wandering.shop. I can get the wandering.shop home feed to load on a desktop or mobile app (Tootdon for iPod; I don't have a smartphone), but not in Chrome or Firefox. I can view my notifications, the local and federated timelines, and a tracked tag.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by mal0ki** [comment]
It's been a few days, are these issues persistent?

**Comment by silverandgray** [comment]
Yes, this issue has been persistent for approximately two months.

**Comment by mal0ki** [comment]
Have you accidentally added some filter in the regex for the home TL? 

![image](https://user-images.githubusercontent.com/28316305/40081357-07759fd6-588e-11e8-888c-f39f752dacfc.png)

**Comment by silverandgray** [comment]
I don't think so. I have "Show boost" and "Show replies" turned on and I am not filtering anything under "Advanced." I am filtering non-English languages from public timelines but the people I follow use English.
